docs/design/features/first-class-structs.md

```diff
@@ -256,7 +256,7 @@ This would be enabled first by [Defer ABI-specific transformations to Lowering](
 
   * Related: #11407, #17257
 
-###  <a name="Improve-Struct-Promotion"/>Improve Struct Promotion
+###  Improve Struct Promotion
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
###  Improve Struct Promotion

```